strawberryfields/result.py

```diff
@@ -80,7 +103,7 @@ def samples(self) -> Optional[np.ndarray]:
         Returned measurement samples will have shape ``(shots, modes)``.
 
         Returns:
-            array[array[float, int]]: measurement samples returned from
+            ndarray[ndarray[float, int]]: measurement samples returned from
```

Comment:
Should this read `np.ndarray` or even `numpy.ndarray`?

Reply:
Here our convention is `ndarray`, or in some cases, `array_like`, to match the numpy documentation

Reply:
e.g., https://numpy.org/doc/stable/reference/generated/numpy.dot.html#numpy.dot